README.md

```diff
@@ -72,7 +72,33 @@ of the calibration is not yet handled by the module:
 echo 0 | sudo tee /sys/bus/wmi/drivers/acer-wmi-battery/calibration_mode
 ```
 
-### Related work
+## Persistent installation (DKMS)
+
+If you found this driver to be working on your laptop, you may want to install it into your system for ease of use.
+
+1) Install DKMS and generic kernel headers (this will always get you the latest headers), on Debian or Ubuntu it can be done with:
+
+```
+sudo apt install dkms linux-headers-generic
```

Comment:
I checked what linux-headers-generic would install on my system and you are right, it depends on the package for the latest kernel headers. But what if you are not running the latest kernel version - as I am. Since the build instructions already describe how to install the headers for the current kernel, I would remove the "linux-headers-generic" from this apt invocation.